code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -287,16 +287,13 @@
 
 
 /mob/living/carbon/flash_act(intensity = 1, override_blindness_check = 0, affect_silicon = 0, visual = 0)
-	if(NOEYES in dna?.species?.species_traits)
-		return
-	var/obj/item/organ/eyes/eyes = getorganslot(ORGAN_SLOT_EYES)
-	if(!eyes) //can't flash what can't see!
-		return
-
 	. = ..()
 
 	var/damage = intensity - get_eye_protection()
 	if(.) // we've been flashed
+		var/obj/item/organ/eyes/eyes = getorganslot(ORGAN_SLOT_EYES)

```

Comment:
why was this moved

Reply:
Ask Calev, I literally just pressed "revert" button on the merge message in the old PR on github site.

Reply:
prob should stay at top even if you're reverting DNA chance. No point in even running inheritance if you're just going to lolnope it imo.

Reply:
Good point, albeit it could be removed entirely - it's never actually reached, get_eye_protection() has a check for eyes too and will fail the check in parent if they're missing.

Reply:
even better